src/components/InvertedFlatList/index.native.js

```diff
@@ -7,7 +7,12 @@ export default forwardRef((props, ref) => (
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
         ref={ref}
-        inverted
         CellRendererComponent={CellRendererComponent}
+        /**
+         * To achieve absolute positioning and handle overflows for list items, the property must be disabled
+         * for Android native builds.
+         * Source: https://reactnative.dev/docs/0.71/optimizing-flatlist-configuration#removeclippedsubviews
+         */
+        removeClippedSubviews={false}
```

Comment:
Why are we merging both ios and android files?
this `removeClippedSubviews={false}` seems to be specific to android currently.

@bernhardoj

Reply:
@bernhardoj Could you share any information on this? I couldn't find the same in docs.